packages/suite/src/components/wallet/TransactionItem/CommonComponents.tsx

```diff
@@ -40,44 +41,19 @@ export const Content = styled.div`
 `;
 
 export const Description = styled(HiddenPlaceholder)`
-    color: ${({ theme }) => theme.legacy.TYPE_DARK_GREY};
-    font-size: ${variables.FONT_SIZE.NORMAL};
-    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    color: ${({ theme }) => theme.textSubdued};
+    ${typography.body};
     line-height: 1.5;
     display: flex;
     justify-content: space-between;
     overflow: hidden;
     white-space: nowrap;
 `;
 
-export const NextRow = styled.div`
-    display: flex;
-    flex: 1;
-    align-items: flex-start;
-    margin-bottom: 6px;
-`;
-
-export const TargetsWrapper = styled.div`
-    display: flex;
-    flex-direction: column;
-    flex: 1;
-    overflow: hidden;
-    padding-right: 10px;
-    margin-right: -10px;
-`;
-
-const amountStyle = css`
-    width: 100%;
-    color: ${({ theme }) => theme.legacy.TYPE_DARK_GREY};
-    font-size: ${variables.FONT_SIZE.NORMAL};
-    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
-    white-space: nowrap;
-`;
-
-export const StyledFormattedCryptoAmount = styled(FormattedCryptoAmount)`
-    ${amountStyle}
-`;
-
-export const StyledFormattedNftAmount = styled(FormattedNftAmount)`
-    ${amountStyle}
-`;
+export const Id = ({ id }: { id?: string }) => {
```

Comment:
Does it belong here?

Reply:
No, since it was used only in one component I moved it there